scalableminds/webknossos
# In stream WS requests, disable timeout

Fixes broken volume downloads.

The only code that uses the WSRequests library for a streaming GET Request is fetching the volume data to webknossos, to pack it into a zip together with an NML

### Steps to test:
- download very large volume tracing, should contain valid data.zip even if download takes longer than 2 minutes
- fake this long download time by adding a sleep in `getData` in `VolumeTracingController.scala'

### Issues:
- fixes #3007 

------
